cluster-scope/overlays/ocp-staging/externalsecrets/moc-sso-client-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: "kubernetes-client.io/v1"
+kind: ExternalSecret
+metadata:
+  name: moc-sso-client-secret
```

Comment:
To match the names of the existing manifests, this should be `sso-clientsecret-moc`.